test/CMakeLists.txt

```diff
@@ -15,6 +15,14 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+# disable fast math for ALL TESTS
+check_cxx_compiler_flag("-fno-fast-math" _fno_fast_math_option)
+if (_fno_fast_math_option)
+    set (CMAKE_CXX_FLAGS "-fno-fast-math ${CMAKE_CXX_FLAGS}")
```

Comment:
Current decision - to switch off "fast math" for all tests.
This decision were discussed with @akukanov